app/theme/client/imports/components/modal.css

```diff
@@ -141,22 +141,15 @@
 	&__footer {
 		display: flex;
 
-		flex: 0 0 auto;
+		padding: 1rem;
 
-		padding: 16px;
+		flex: 0 0 auto;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "flex" to come before "padding" (order/properties-order)](https://app.codacy.com/app/RocketChat/Rocket-Chat/pullRequest?prid=4651020)